<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>packet_filter</class>
 <widget class="QDialog" name="packet_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_no">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>选择需要过滤的协议</string>
   </property>
   <widget class="QCheckBox" name="is_TCP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_ARP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_ICMP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ICMP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_UDP">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_RARP">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RARP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_IGMP">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>IGMP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_myself">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>自定义</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_filter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;http port:80&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
